Bottom: a904f2cc1619b45e8464e3fe446175ea6783cf2d
Top:    b08c9b0bd2804799e21b405b08d146234957cde8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 13:55:18 +0000

Refresh of extend-object3d_from_contours

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a65ed31..bcd87e6 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1695,7 +1695,8 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               -depth - HACK_COPPER_THICKNESS, /* Top */
 #endif
                               copper_appearance,
-                              NULL));
+                              NULL,
+                              false)); /* Don't invert */
 
   }
